<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trova i file più grandi nel PC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Trova i file e le cartelle più grandi nel PC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDirChooser">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cambia percorso..</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>891</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelPathDesc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>159</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Percorso di lavoro: </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>1021</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status: </string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>1121</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>891</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRefresh">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>40</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Aggiorna</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelKeys">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>File selezionato: </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelKeysPath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>1121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDelete">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>520</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Elimina file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSelected">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>1111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonFolder">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>520</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Apri percorso file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonWrite">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>520</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Scrivi risultati su file</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
